list_openstack_entities.py

```diff
@@ -0,0 +1,132 @@
+#!/bin/python
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Refer to the README and COPYING files for full details of the license
+
+import getopt
+import json
+import logging
+import openstack
+import sys
+import yaml
+
+
+logging.basicConfig(level=logging.ERROR, filename='openstack_client.log')
+
+CLOUD_CONFIG_NAME='ovirt'
+DEFAULT_AUTH_URL='https://engine-test:35357/v2.0/'
+DEFAULT_USER='admin@internal'
+DEFAULT_PASSWORD='admin'
+DEFAULT_VERIFY=False
+CA_PATH='/etc/pki/ovirt-engine/ca.pem'
+
+
+class CloudConfigurationError(Exception):
```

Comment:
I don't care as much as the code makes it look like ... :) 

I can drop all that and use the openstack SDK configuration files loading mechanism; that would 
make the tool be invoked by.
```bash
OS_CLIENT_CONFIG_FILE=playbooks/clouds.yml python list_openstack_entities.py -p
```

Is there another way to 'feed' it the clouds.yml ? From my understanding, that is the correct way since in the [openstack SDK documentation](https://docs.openstack.org/openstacksdk/latest/user/config/configuration.html#config-files) the *clouds.yml* file is looked after in:
* Current Directory
* ~/.config/openstack
* /etc/openstack

Remember that the clouds.yml file is currently in $REPO_ROOT/playbooks/clouds.yml whereas the python script is in $REPO_ROOT/list_python_entities.py

I could copy the clouds.yml file to any of those other dirs though. Your opinion is highly valued here.

Reply:
From my point of view, any simple example of how to call the script and the link to OpenStack SDK documentation is enough, because the configuration of the OpenStack clients is documented in many places on the web.